### PR TITLE
Add person and move details in right hand rail of PER views [P4 1992]

### PR DESCRIPTION
**Pull request by solidgoldpig**
### What changed

Adds person and move details to sidebar on PER overview, section and step pages.

### Why did it change

Missing feature - always part of design

### Issue tracking

- [P4-1992 - PER: Add person and move details in right hand rail](https://dsdmoj.atlassian.net/browse/P4-1992)

## Screenshots

| After | Before |
| ------ | ----- |
| <kbd>Person_Escort_Record_for_MCGLYNN__CLAIRE_–_Book_a_secure_move</kbd> | <kbd>Person_Escort_Record_for_MCGLYNN__CLAIRE_–_Book_a_secure_move</kbd> |
| <kbd>Risk_information_overview_–_Person_Escort_Record_–_Book_a_secure_move</kbd> | <kbd>Risk_information_overview_–_Person_Escort_Record_–_Book_a_secure_move</kbd> |
| <kbd>Security_–_Risk_information_–_Person_Escort_Record_–_Book_a_secure_move</kbd> | <kbd>Security_–_Risk_information_–_Person_Escort_Record_–_Book_a_secure_move</kbd> |
| <kbd>Provide_sensitive_medical_details_–_Health_information_–_Person_Escort_Record_–_Book_a_secure_move</kbd> | <kbd>Provide_sensitive_medical_details_–_Health_information_–_Person_Escort_Record_–_Book_a_secure_move</kbd> |

#### Automated testing

- [x] Added unit tests to cover changes
- [ ] Added end-to-end tests to cover any journey changes

#### Manual testing

Has been tested in the following browsers:

- [x] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Internet Explorer

### Environment variables

- [x] No environment variables were added or changed

### Other considerations

- [x] No new [Personally Identifiable Information (PII)](https://support.google.com/analytics/answer/6366371?hl=en) is being sent via analytics

### CODE REVIEW

File: common/helpers/move/get-can-cancel-move.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+function getCanCancelMove(move, userPermissions = []) {
+  // const canCancelMove =
```
Comment (teneightfive) [inline common/helpers/move/get-can-cancel-move.js:2]: Rogue comments?

**Review by teneightfive — APPROVED** [review]

File: common/middleware/set-move-with-summary.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+function setMoveWithSummary(req, res, next) {
+  res.locals = {
+    ...moveHelpers.getMoveWithSummary(req.move),
+    ...res.locals,
+  }
+  next()
+}
+
+module.exports = setMoveWithSummary
```
Comment (merlinc) [inline common/middleware/set-move-with-summary.js:11]: Sorry, I think I've missed where this common middleware is being used?

File: common/controllers/framework/framework-step.js [hunk 3]
```diff
@@ -78,6 +79,7 @@ class FrameworkStepController extends FormWizardController {
     this.use(this.setSyncStatusBanner)
     this.use(this.setPrefillBanner)
     this.use(this.seti18nContext)
+    this.use(setMoveWithSummary)
```
Comment (merlinc) [inline common/controllers/framework/framework-step.js:82]: Aha, here it is!

**Review by merlinc — APPROVED** [review]

File: common/middleware/set-move-with-summary.js (hunk 2 above)
Comment (teneightfive) [reply common/middleware/set-move-with-summary.js:11]: Could this live in `/common/middleware/framework/` seeing as its related to the framework controllers?

**Review by teneightfive — APPROVED** [review]
Looks good. I would say though that the commits felt a little _too_ atomic. I think it could have been easier to run through them by grouping things like the creation of the include alongside actually swapping the previous code out.

File: common/middleware/set-move-with-summary.js (hunk 2 above)
Comment (solidgoldpig) [reply common/middleware/set-move-with-summary.js:11]: I put it there... but it's not technically got anything to do with the framework per se so moved it to the `/common/middleware` jumble. Maybe a move directory is waving to us over the horizon?

**Pull request merged**